unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/Tests/WearablesVisualTests.cs

```diff
@@ -21,9 +22,12 @@ protected override IEnumerator SetUp()
     {
         yield return base.SetUp();
 
+        combiner = new AvatarMeshCombinerHelper();
+        combiner.uploadMeshToGpu = false;
```

Comment:
can this two settings be as a part of `AvatarMeshCombinerHelper` constructor?

Reply:
Is not that easy beacause they must be part of the interface so we can mock the combining in tests.

The `AvatarMeshCombinerHelper` requires a small refactor to make this doable.

Reply:
yeah, i've been checking the code and it makes more sense to have this properties in the interface